distro/configs/superset/superset-init.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+STEP_CNT=6
+
+echo_step() {
+cat <<EOF
+######################################################################
+Init Step ${1}/${STEP_CNT} [${2}] -- ${3}
+######################################################################
+EOF
+}
+# Initialize the database
+echo_step "1" "Starting" "Applying DB migrations"
+superset db upgrade
+echo_step "1" "Complete" "Applying DB migrations"
+
+# Create an admin user
+echo_step "2" "Starting" "Setting up admin user ( admin / $ADMIN_PASSWORD )"
+superset fab create-admin \
+              --username admin \
```

Comment:
Could we consider enabling configuring of the admin username through environment variable?

Reply:
Is it happening only with Superset?

Reply:
> Is it happening only with Superset?

Yes, only Superset.